tests/xtimer_usleep/main.c

```diff
@@ -24,35 +26,77 @@
 
 #include "xtimer.h"
 #include "timex.h"
+#include "stdlib.h"
 
 #define RUNS                (5U)
 #define SLEEP_TIMES_NUMOF   (sizeof(sleep_times) / sizeof(sleep_times[0]))
 
-static const uint32_t sleep_times[] = { 10000, 50000, 100000 };
+static const uint32_t sleep_times[] = { 10000, 50000, 1234, 5678, 121212, 98765, 75000};
 
+#define ERROR_US 500
```

Comment:
Errors of more than 500 microseconds are way too big IMHO. `xtimer_usleep()` is used for very short periods of time. If your hardware doesn't support it, it must be blacklisted for this test (e.g. running a very low speed timer).